automation/TaskManager.py

```diff
@@ -253,8 +245,7 @@ def _manager_watchdog(self) -> None:
 
             # Check for browsers or displays that were not closed correctly
             # 300 second buffer to avoid killing freshly launched browsers
-            # TODO This buffer should correspond to the maximum spawn timeout
-            if self.process_watchdog:
+            if self.manager_params[PROCESS_WATCHDOG]:
```

Comment:
```suggestion
             # TODO This buffer should correspond to the maximum spawn timeout
            if self.manager_params[PROCESS_WATCHDOG]:
```

This TODO still exists. The one that can be removed is further up.

Reply:
I have removed it in latest push 12 mins ago. Is this possible that you may have check outdated commit? Please let me know if it is still there in latest commit

Reply:
I just refreshed and GitHub claims it's still present in the latest version of this PR.

Reply:
Okay, I will commit this suggestion.

Reply:
Can you commit this change again? Seems like it got lost along the way.